Sources/StringsLintFramework/Parsers/StructuredPlaceholderCommentParser.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  StructuredPlaceholderCommentParser.swift
+//  
+//
+//  Created by Mark Hall on 2022-07-19.
+//
+
+import Foundation
+
+public struct StructuredPlaceholderCommentParser: LocalizableParser {
```

Comment:
I think we need a better name for this file. It always get me confused and makes me think this is the rule and stringsdict treatment should be in this file